addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+    - If you are using your application source, replace `source.git.url` & `source.git.revision`.
+   > Note: To use a private git repo follow the instructions in [secrets](secrets.md)
+
+   Apply that image resource to the cluster
+
+    ```bash
+    kubectl apply -f image.yaml
+    ```
+
+   You can now check the status of the image resource.
+
+   ```bash
+   kubectl -n default get images
+   ```
+
+   You should see that the image resource has an unknown READY status as it is currently building.
+
+   ```
+    NAME                  LATESTIMAGE   READY
+    tutorial-image                      Unknown
+    ```
+
+   You can tail the logs the image that is currently building using the [kp cli](https://github.com/vmware-tanzu/kpack-cli/blob/main/docs/kp_build_logs.md)
+
+    ```
+    kp build logs tutorial-image -n default
+    ``` 
+
+   Once the image resource finishes building you can get the fully resolved built OCI image with `kubectl get`
+
+    ```
+    kubectl -n default get image tutorial-image
+    ```  
+
+   The output should look something like this:
+    ```
+    NAMESPACE   NAME                  LATESTIMAGE                                        READY
+    default     tutorial-image        index.docker.io/your-project/app@sha256:6744b...   True
+    ```
+
+   The latest OCI image is available to be used locally via `docker pull` and in a kubernetes deployment.
+
+8. Run the built app locally
+
+   Download the latest OCI image available in step #6 and run it with docker.
+
+   ```bash
+   docker run -p 8080:8080 <latest-image-with-digest>
+   ```
+
+   You should see the java app start up:
+   ```
+       
+              |\      _,,,--,,_
+             /,`.-'`'   ._  \-;;,_
+    _______ __|,4-  ) )_   .;.(__`'-'__     ___ __    _ ___ _______
+    |       | '---''(_/._)-'(_\_)   |   |   |   |  |  | |   |       |
+    |    _  |    ___|_     _|       |   |   |   |   |_| |   |       | __ _ _
+    |   |_| |   |___  |   | |       |   |   |   |       |   |       | \ \ \ \
+    |    ___|    ___| |   | |      _|   |___|   |  _    |   |      _|  \ \ \ \
+    |   |   |   |___  |   | |     |_|       |   | | |   |   |     |_    ) ) ) )
+    |___|   |_______| |___| |_______|_______|___|_|  |__|___|_______|  / / / /
+    ==================================================================/_/_/_/
+    
+    :: Built with Spring Boot :: 2.2.2.RELEASE
+   ``` 
+
+9. kpack rebuilds
+
+   We recommend updating the kpack image resource with a CI/CD tool when new commits are ready to be built.
+   > Note: You can also provide a branch or tag as the `spec.git.revision` and kpack will poll and rebuild on updates!
+
+   We can simulate an update from a CI/CD tool by updating the `spec.git.revision` on the image resource used in step #6.
+
+   If you are using your own application please push an updated commit and use the new commit sha. If you are using Spring Pet Clinic you can update the revision to: `4e1f87407d80cdb4a5a293de89d62034fdcbb847`.
```

Comment:
remove 'please' as per [style guide](https://tanzucommunityedition.io/docs/latest/contribute/style-guide/#use-simple-and-direct-language)